leaf-ui/js/displayAnalysis.js

```diff
@@ -346,19 +351,44 @@ function addAnalysisConfig(config) {
     mainElement = document.getElementById(config.id);
     mainElement.querySelector('.config-elements').addEventListener('dblclick', function(){rename(this /** Config element */)});
     mainElement.querySelector('.config-elements').addEventListener('click', function(){switchConfigs(this.parentElement /** Config element */)});
-    mainElement.querySelector('.dropdown-button').addEventListener('click', function(){toggleDropdown(this.parentElement.parentElement /** Config element */)})
+    mainElement.querySelector('.dropdown-button').addEventListener('click', function(){toggleDropdown(this.parentElement.parentElement /** Config element */)});
+    mainElement.querySelector('.deleteconfig-button').addEventListener('click', function(){ 
+        try{removeConfiguration(this.parentElement.parentElement)/** Config element */} 
+        catch(e){console.error(e); alert(e);}
+    });
+
+    //Whenever first or loaded configs are added, update the highest position
+    highestPosition = config.initialPosition;
```

Comment:
I made a fix to prevent duplicate names but would love to replicate this bug so I can see how it happened/how to fix.

Reply:
Yes for sure! Its actually cause by saving and loading. So to replicate you would need to create some configs, rename any config that is not the last config, and then save. When you reload, since it saves/loads in order of most recently edited, it will load the last edited config (aka the config you renamed) last. Before you made the check this meant that the highest position would actually just be the initial position of the last loaded config, not necessarily the highest initial position. For example if you had config1, config2, config3, and then edited config2 to be named something else (lets say prev2), it would reload in the order config1, config3, prev2. So highest position would be saved as 2 from prev2, and the next config added would be named config3 and be a duplicate. Sorry I know that is a lot of words so let me know if it doesn't make sense!